yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -171,24 +171,11 @@ export class AvmPersistableStateManager {
    * @returns exists - whether the message exists in the L1 to L2 Messages tree
```

Comment:
How is this opcode supposed to handle nullified messages? See comment in https://github.com/AztecProtocol/aztec-packages/pull/5805

Reply:
As mentioned, I don't think exists should necessarily take into account if it is "active", just that it is in the tree. Separately, you might not be able to even know if it is nullified because you might not know the secret 🤷